networklibrary/src/main/java/com/eis/communication/network/commands/RemoveResource.java

```diff
@@ -2,37 +2,33 @@
 
 import androidx.annotation.NonNull;
 
-import com.eis.communication.network.Command;
 import com.eis.communication.network.NetDictionary;
 
-
 /**
  * Command to remove a resource from the network dictionary
```

Comment:
```suggestion
 * Command to remove a resource from the local network dictionary
```

Reply:
After almost a month I forgot what a network dictionary actually is so let's make it clear.